src/plugins/data/public/search/session/sessions_client.ts

```diff
@@ -37,7 +36,7 @@ export class SessionsClient {
     this.http = deps.http;
   }
 
-  public get(sessionId: string): Promise<SavedObject<BackgroundSessionSavedObjectAttributes>> {
```

Comment:
@Dosant, you'll noticed I removed references to the `BackgroundSession` stuff here since it was moved to xpack. The corresponding routes are also now only available in Basic+, which probably means this file would make more sense being moved to xpack, but I'm not exactly sure how you want to go about it. Do you have any thoughts about it?

Reply:
I think we should move those to x-pack too

Reply:
I think we can move `session_client` implementation into x-pack but will have to leave interface in OSS (decoupled from `BackgroundSessionSavedObjectAttributes`). 

But I don't think we should move `sessionService` into x-pack because of how OSS apps rely on it.